Jenkinsfile

```diff
@@ -40,5 +40,5 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipelineDAOS(['distros':['centos8','centos7'],
-                             'rpmlint_check': false])
+packageBuildingPipeline(['distros':['centos8','centos7'],
+                         'rpmlint_check': false])
```

Comment:
Why are we NOOPing the rpmlint check?  Have you tried re-enabling this since we have moved building to Fedora?

Reply:
No we are not.  And while that post-rpm lint typically caused quite a bit of work, i think ultimately it was to good benefit.

It would be interesting some day to see what tool they were using for that and utlise it ourselves.  That said, I'm not willing to go back to `build` for it.  `mock` is just a better tool.